packages/mobile/src/verify/VerificationInputScreen.tsx

```diff
@@ -115,7 +117,7 @@ function HeaderLeftButton() {
 }
 
 class VerificationInputScreen extends React.Component<Props, State> {
-  static navigationOptions = ({ navigation }: ScreenProps) => ({
+  static navigationOptions = ({ navigation, route }: ScreenProps) => ({
```

Comment:
I think it's unnecessary to use `route` here if you already have `chouseToRestoreAccount` in state